tests/System.Buffers.Experimental.Tests/MemoryTests.cs

```diff
@@ -63,12 +62,12 @@ public unsafe void PinnedArrayMemoryLifetime()
             }
         }
 
-        static void TestLifetime(OwnedBuffer<byte> owned)
+        static void TestLifetime(OwnedMemory<byte> owned)
         {
-            Buffer<byte> copyStoredForLater;
+            Memory<byte> copyStoredForLater;
             try {
-                Buffer<byte> memory = owned.Buffer;
-                Buffer<byte> memorySlice = memory.Slice(10);
+                Memory<byte> memory = owned.AsMemory;
```

Comment:
AsMemory should not be a property. It should be either a method or should be called Memory

Reply:
I agree.
I can make this change, but I was waiting for API review:
https://github.com/dotnet/coreclr/issues/13801#issuecomment-327525698
> Before we do this change, can we discuss (at the API review) when we intend to use properties and when methods for span/memory related conversions? I